src/main/java/org/zalando/nakadi/validation/schema/CompatibilityModeChangeConstraint.java

```diff
@@ -14,7 +14,7 @@
     final Map<CompatibilityMode, List<CompatibilityMode>> allowedChanges = ImmutableMap.of(
             CompatibilityMode.COMPATIBLE, Lists.newArrayList(CompatibilityMode.COMPATIBLE),
             CompatibilityMode.FORWARD, Lists.newArrayList(CompatibilityMode.FORWARD, CompatibilityMode.COMPATIBLE),
-            CompatibilityMode.NONE, Lists.newArrayList(CompatibilityMode.NONE)
+            CompatibilityMode.NONE, Lists.newArrayList(CompatibilityMode.NONE, CompatibilityMode.FORWARD)
```

Comment:
I'm not intimate with the code base so forgive me for asking but does this include the migration from "none" to "compatible" which should also be possible in my opinion?